assets/stylesheets/shared/_themes.scss

```diff
@@ -0,0 +1,97 @@
+// ==========================================================================
+// Themes
+//
+// Theming variables and color scheme definitions
+//
+// 1.0 Theme variables
+// 2.0 Color scheme definitions
+// 3.0 Theming classes
+//
+// Usage:
+//
+// Add a new theme variable:
+// I.) Add a new theme variable to section 1.0 of this file
+// II.) Add new custom property matching the theme variable name to $color-schemes > default
+// III.) Add the same custom property to each theme in $color-schemes
+//
+// Add a new theme:
+// I.) Add a new block to $color-schemes by copying $color-schemes > default
+// II.) Change name of new color scheme to desired name for the new theme
+// III.) Change color values assigned to the custom properties of the new theme
+// ==========================================================================
+
+/**
+ * 1.0 Theme variables
+ *
+ * Declare theme variables and assign default colors
+ */
+
+// Layout
+$masterbar-color:                  $white;
+$masterbar-bg-color:               $blue-wordpress;
+$masterbar-border-color:           darken( $masterbar-bg-color, 4% );
+$masterbar-item-active-bg-color:   darken( $masterbar-bg-color, 17% );
+$masterbar-item-hover-bg-color:    lighten( $masterbar-bg-color, 5% );
+$masterbar-item-new-color:         $blue-wordpress;
```

Comment:
Ideally the color shouldn't be assigned per-component like this, but just override the individual variables. Is this needed just in the scope of this PR targeting just the masterbar, and will be removed later?